src/mixins/helpers.js

```diff
@@ -15,9 +15,9 @@ var helpers = {
     var slideWidth;
 
     if (!props.vertical) {
-      slideWidth = trackWidth/props.slidesToShow;
+      slideWidth = this.getWidth(ReactDOM.findDOMNode(this))/props.slidesToShow;
```

Comment:
@Josh-ES I think, it should be ReactDOM.findDOMNode(this.track) instead of ReactDOM.findDOMNode(this)